coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/CalciteTrinoUDFMap.java

```diff
@@ -36,6 +36,7 @@ private CalciteTrinoUDFMap() {
     createUDFMapEntry(UDF_MAP, SqlStdOperatorTable.ITEM, 2, TrinoElementAtFunction.INSTANCE);
 
     // Math Functions
+    createUDFMapEntry(UDF_MAP, hiveToCalciteOp("pmod"), 2, "mod");
```

Comment:
I have encountered the same problem and try your solution, but mod and pmod are not equivalent.
For example:
`select pmod(-9, 4)` is `3` in hive 
`select mod(-9, 4)` is `-1` in presto
So I implemented `pmod` in presto.